janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java

```diff
@@ -57,4 +62,36 @@ public boolean supportsWildcardQuery() {
     protected boolean supportsCollections() {
         return false;
     }
+
+    @Test
+    public void testPrintSchemaElements() {
```

Comment:
I know JanusGraph has lots of big tests that often test all aspects of a certain functionality in a single test, but I think it's more helpful to have smaller tests where each one only tests one aspect. That makes it easier to understand what's wrong as the failed test won't be named `testPrintSchemaElements` but something like `testPrintVertexLabels` (assuming that each assert here gets its own test) or even `printVertexLabelsShouldOnlyIncludeHeadlineIfNoVertexLabelsDefined`.

Reply:
I had it split out initially, but wanted to only call GraphOfTheGodsFactory.load() once for all the tests. I was thinking about using `@Before` for the graph setup, but didn't want to affect the other tests in the super class. I'll take another look to see If I can group the setup without affecting the other tests.

Reply:
A `@Before` statement with conditionals to specify which test to run against will get evaluated by ~2500 lines of inherited tests, due to the way it's structured right now. The other option that now comes to me after reviewing 2000 lines of junit5 updates is to set it up as a Parameterized test.  It'll have timing implications with merging @farodin91's PR and I'll have to rewrite it when I merge to master, but I'll take a crack at switching it over.

Reply:
Ah, I didn't think of the high setup costs here. Then it makes of course more sense to use just one test here.
Wouldn't a parameterized test have the same problem?

Reply:
In junit5 you don't have to parameterize the entire class you can just specify `@ParametrizedTest` instead of `@Test`. I can initialize the graph in the method source for the test and then add the output Strings to the argument, and then do a single cleanup. It should end up a lot cleaner and easier to manage as I add new info, like schema constraints.

Reply:
OK, thanks for the explanation @chupman! That sounds really like a good solution then.